src/main/java/com/aws/greengrass/deployment/DeploymentService.java

```diff
@@ -323,6 +324,9 @@ private void finishCurrentDeployment() throws InterruptedException {
         String configurationArn = currentDeploymentTaskMetadata.getConfigurationArn();
         DeploymentType type = currentDeploymentTaskMetadata.getDeploymentType();
         List<String> rootPackages = currentDeploymentTaskMetadata.getRootPackages();
+        DeploymentComponentUpdatePolicyAction componentUpdatePolicy =
```

Comment:
do you need to check if `getComponentUpdatePolicy` returns null in these three places?

Reply:
Yeah, but I don't see how could be set to null in any of the deployment types. Thing and thing group deployments force the customer to select either notify or skip and we hard code for local deployments.
If I do catch a null update policy, what should be the default?